### DISCUSSION TITLE
My PullRequest (translation of documentation) broke units tests...

### DISCUSSION THREAD

**Discussion by axel584** [fastapi/fastapi#6149] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

I translated a page of the documentation in french, and make a Pull Request : 
https://github.com/tiangolo/fastapi/pull/5477

And the CI/CD shows 2 problems with the Unit Test (for the 3.7 and 3.9 plateform) : 
https://github.com/tiangolo/fastapi/actions/runs/3215376279/jobs/5256425256
https://github.com/tiangolo/fastapi/actions/runs/3215376279/jobs/5256425208

I don't understand why (I just add .md file).

The log explains : 

```
        modulename = "anyio._backends._" + asynclib_name
        try:
>           return sys.modules[modulename]
E           KeyError: 'anyio._backends._trio'

/opt/hostedtoolcache/Python/3.9.14/x64/lib/python3.9/site-packages/anyio/_core/_eventloop.py:153: KeyError
```

Is it a missing module in the runner of the CI/CD ? To be honest, it's outside of my skills... Thanks for your help.

### Operating System

Other

### Operating System Details

the runner of the CI/CD : 
https://github.com/tiangolo/fastapi/actions/runs/3215376279/jobs/5256425256
https://github.com/tiangolo/fastapi/actions/runs/3215376279/jobs/5256425208

### FastAPI Version

0.85.0

### Python Version

3.7 and 3.9

**Comment by carlosm27** [comment]
Hey @axel584,  thank you raise this issue. I have the same problem with [pull request 5466](https://github.com/tiangolo/fastapi/pull/5466). It is a Spanish translation and test 3.7 and test 3.9 fail. And I don't have any idea about the causes.

**Comment by mbroton** [comment, marked as answer]
Hi, the cause of these problems is identified and fixed here: https://github.com/tiangolo/fastapi/pull/5456
But it's not merged yet 😔

**Comment by axel584** [comment]
@mbroton : Thank you very much. I hope your PR will merge soon.

**Comment by Xewus** [comment]
It seems, this problem (or very like) has moved to Python 3.9 - 3.10 .
I see it here [#5336](https://github.com/tiangolo/fastapi/pull/5336)  (because  its mine  :blush:)
and for another one [#5550](https://github.com/tiangolo/fastapi/pull/5550) .
Unfortunately, I do not understand this error :sheep:

**Comment by iudeen** [comment]
The failure is due to trio warning. Possible side effect of not pinning test deps.

**Comment by Xewus** [comment]
Is it my fault?
I attached the translation of only one page and added one line for it in `docs/ru/docs/deployment/index.md` .
And last month there were no problems with this.
Earlier translations were fine too...

**Comment by iudeen** [comment]
No, not your fault.
